build/Build.cs

```diff
@@ -273,10 +273,10 @@ void PublishCalamariProjects(List<Project> projects)
             var packagesToPublish = crossPlatformPackages.Concat(netFxPackages);
 
             packagesToPublish.ForEach(PublishPackage);
-            Task.WhenAll(SignDirectoriesTasks);
+            await Task.WhenAll(SignDirectoriesTasks);
             
             projects.ForEach(CompressCalamariProject);
-            Task.WhenAll(ProjectCompressionTasks);
+            await Task.WhenAll(ProjectCompressionTasks);
```

Comment:
It _seems_ to me that we should be only compressing the projects _after_ we've signed the directories 🤔 

as we weren't awaiting the `Task.WhenAll`'s, the compress would always run immediately without waiting for the signing to complete